tracing/config/tracing.go

```diff
@@ -139,3 +140,43 @@ func NewTracingConfigFromMap(cfgMap map[string]string) (*Config, error) {
 func NewTracingConfigFromConfigMap(config *corev1.ConfigMap) (*Config, error) {
 	return NewTracingConfigFromMap(config.Data)
 }
+
+// JsonToTracingConfig converts a json string of a Config.
+// Returns a non-nil Config always and an eventual error.
+func JsonToTracingConfig(jsonCfg string) (*Config, error) {
+	if jsonCfg == "" {
+		return defaultConfig(), errors.New("empty json tracing config")
+	}
+
+	var configMap map[string]string
+	if err := json.Unmarshal([]byte(jsonCfg), &configMap); err != nil {
+		return defaultConfig(), err
+	}
+
+	cfg, err := NewTracingConfigFromMap(configMap)
+	if err != nil {
+		return defaultConfig(), nil
+	}
+	return cfg, nil
+}
+
+// TracingConfigToJson converts a Config to a json string.
+func TracingConfigToJson(cfg *Config) (string, error) {
+	if cfg == nil {
+		return "", nil
+	}
+
+	out := make(map[string]string, 5)
+	out[backendKey] = string(cfg.Backend)
+	if cfg.ZipkinEndpoint != "" {
+		out[zipkinEndpointKey] = cfg.ZipkinEndpoint
+	}
+	if cfg.StackdriverProjectID != "" {
+		out[stackdriverProjectIDKey] = cfg.StackdriverProjectID
+	}
+	out[debugKey] = strconv.FormatBool(cfg.Debug)
+	out[sampleRateKey] = strconv.FormatFloat(cfg.SampleRate, 'f', -1, 64)
```

Comment:
I find `Sprint`  easier to read, especially for the float case.
```suggestion
	out[debugKey] = fmt.Sprint(cfg.Debug)
	out[sampleRateKey] = fmt.Sprint(cfg.SampleRate)
```